app/src/main/java/org/simple/clinic/summary/teleconsultation/status/TeleconsultStatusModel.kt

```diff
@@ -1,14 +1,17 @@
 package org.simple.clinic.summary.teleconsultation.status
 
 import org.simple.clinic.teleconsultlog.teleconsultrecord.TeleconsultStatus
+import java.util.UUID
 
 data class TeleconsultStatusModel(
+    val teleconsultRecordId: UUID,
     val teleconsultStatus: TeleconsultStatus?
 ) {
 
   companion object {
 
-    fun create(teleconsultStatus: TeleconsultStatus?) = TeleconsultStatusModel(
+    fun create(teleconsultRecordId: UUID, teleconsultStatus: TeleconsultStatus?) = TeleconsultStatusModel(
```

Comment:
We've mixed two changes in this commit:

- Adding the teleconsult record ID as a property of the model
- Handling the done click event

These should be two separate commits for easier review.